app/views/users/profiles/show.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1= @profile.user.name + ' ' + t('profile')
```

Comment:
you can have variables in yml. 
for example:
 `t('.hello', name: current_user.name)`
 in yml: `hello: "Hello, %{name}!"`